cmd/kubelet/app/server.go

```diff
@@ -251,6 +255,12 @@ is checked every 20 seconds (also configurable with a flag).`,
 				klog.InfoS("unsupported configuration:KubeletCgroups is not within KubeReservedCgroup")
 			}
 
+			if kubeletFlags.FailCgroupV1 {
```

Comment:
this should be kubeletConfig (or else we check kubeletFlags twice and never check the kubeletConfig)

Reply:
Sorry pushed the branch before addressing that. Let me fix it. Thanks.